test/dotnet-list-proj.Tests/GivenDotnetListProj.cs

```diff
@@ -0,0 +1,141 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using FluentAssertions;
+using Microsoft.DotNet.Cli.Sln.Internal;
+using Microsoft.DotNet.Tools.Test.Utilities;
+using System;
+using System.IO;
+using System.Linq;
+using Xunit;
+
+namespace Microsoft.DotNet.Cli.List.Proj.Tests
+{
+    public class GivenDotnetListProj : TestBase
+    {
+        [Theory]
+        [InlineData("--help")]
+        [InlineData("-h")]
+        public void WhenHelpOptionIsPassedItPrintsUsage(string helpArg)
+        {
+            var cmd = new DotnetCommand()
+                .ExecuteWithCapturedOutput($"list projects {helpArg}");
+            cmd.Should().Pass();
+            cmd.StdOut.Should().Contain("Usage");
```

Comment:
If localization comes in play here, then it will be a problem for a small string or a big one, right? Plus, these tests should run on english.